packages/google_maps_flutter/google_maps_flutter_platform_interface/test/method_channel/method_channel_google_maps_flutter_test.dart

```diff
@@ -68,5 +81,46 @@ void main() {
         'map#takeSnapshot',
       ]);
     });
+    test('markers send drag event to correct streams', () async {
+      const int mapId = 1;
+      final jsonMarkerDragStartEvent = <dynamic, dynamic>{
+        "mapId": mapId,
+        "markerId": "drag-start-marker",
+        "position": <double>[1.0, 1.0]
+      };
+      final jsonMarkerDragEvent = <dynamic, dynamic>{
+        "mapId": mapId,
+        "markerId": "drag-marker",
+        "position": <double>[1.0, 1.0]
+      };
+      final jsonMarkerDragEndEvent = <dynamic, dynamic>{
+        "mapId": mapId,
+        "markerId": "drag-end-marker",
+        "position": <double>[1.0, 1.0]
+      };
+
+      final MethodChannelGoogleMapsFlutter maps =
+          MethodChannelGoogleMapsFlutter();
+      maps.ensureChannelInitialized(mapId);
+
+      final StreamQueue<MarkerDragStartEvent> markerDragStartStream =
+          StreamQueue(maps.onMarkerDragStart(mapId: mapId));
+      final StreamQueue<MarkerDragEvent> markerDragStream =
+          StreamQueue(maps.onMarkerDrag(mapId: mapId));
+      final StreamQueue<MarkerDragEndEvent> markerDragEndStream =
+          StreamQueue(maps.onMarkerDragEnd(mapId: mapId));
+
+      await sendPlatformMessage(
+          mapId, "marker#onDragStart", jsonMarkerDragStartEvent);
+      await sendPlatformMessage(mapId, "marker#onDrag", jsonMarkerDragEvent);
+      await sendPlatformMessage(
+          mapId, "marker#onDragEnd", jsonMarkerDragEndEvent);
+
+      expect((await markerDragStartStream.next).value.value,
+          equals("drag-start-marker"));
```

Comment:
can you `equals(MarkerId('drag-start-marker'))`?

Reply:
I've done this but I'm going to remove revert it as it's in the platform_interface